scripts/algokit-libraries.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# SETUP
+PATH_TO_SOURCE_REPOS=$1 
```

Comment:
Can this be refactored a little bit to be a bit more modular/maintanable?

This might benefit from being re written in python given that there are already scripts on this repo that has CI and deal with propagating latest docs via python scripts that format the docs. 

If this is further refined would be very handy to at least have a manual workflow trigger for a separate pipeline to execute this considering there are often new docs and releases on algokit - managing each doc change via manual script and prs would be very hard to manage. 

I think there are these main stages done in this script:

clean_existing_content
create_directories
copy_and_modify_content
create_pages
cleanup

that should have dedicated methods. Whether or not to rewrite this to python completely is optional suggestion but ideally this script has to be executable via ci (doesn't have to be fully automated) just a manual workflow trigger that auto opens a pr (similar to how existing workflows are configured on this repo)